src/components/Book.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => (
+  <tr>
+    <td>
+      {' '}
+      {book.id}
+      {' '}
+    </td>
+    <td>
+      {' '}
+      {book.title}
+      {' '}
+    </td>
+    <td>
+      {' '}
+      {book.category}
+      {' '}
+    </td>
+  </tr>
+);
+
+Book.propTypes = {
+  book: PropTypes.func.isRequired,
```

Comment:
- [ ] Why `book` is defined as a `func` here? Isn't supposed to be an object?
![Screenshot from 2020-08-10 14-52-08](https://user-images.githubusercontent.com/30977845/89814377-6ac23800-db19-11ea-889e-8da8107c3830.png)

Reply:
Hi @evansinho @addod19 following this review, I see that the change for this was not made, is there a reason why Book is being checked as a func and not an object? Please leave or comment or change it if it was an oversight :slightly_smiling_face: